### DISCUSSION TITLE
normalized counts ouput

### DISCUSSION THREAD

**Discussion by mannekim** [igordot/sns#19] [Q&A] [answered]
Hi, this is not really an issue but more of a question.
For the normalized counts ouput, is it in RPKM?

**Comment by igordot** [comment, marked as answer]
The normalized counts (`counts.normalized.csv` or `counts.normalized.xlsx`) are scaled by size factors. This is the standard DESeq2 normalization. FPKMs are available as well (`counts.fpkm.csv` or `counts.fpkm.xlsx`).
